### ISSUE TITLE
version 1.1.0 takes antiSMASH4 result?

### ISSUE THREAD

**Issue by uniqiqi** [medema-group/BiG-SCAPE#10]
Hi, I am wondering if BiG-SCAPE 1.1.0 can take the BGC files from antiSMASH 5.1 instead of 6.0? I want to use the MiBIG 2.0 reference dataset from version 1.1.0 but do not want to rerun my data with antiSAMSH 6.0. Thanks!

**Comment by jorgecnavarrom** [comment]
Yes, we didn't remove the types from previous versions of antiSMASH, so it should work ok with 5.1 input
